src/utils.ts

```diff
@@ -65,18 +72,21 @@ export function ensureUUID(thing: Thing): Thing {
 }
 
 export function createThingWithUUID(): Thing {
-  return createThing({ url: uuidUrn() });
+  return createThing({ url: newUuidUrn() });
 }
 
 export function slugToUrl(
   resourceOrUrl: SolidDataset | UrlString,
   slug: Slug
-): UrlString | null {
+): UrlString {
   const url =
     typeof resourceOrUrl === 'string'
       ? resourceOrUrl
       : getSourceUrl(resourceOrUrl);
-  return url && new URL(`#${slug}`, url).toString();
+  return url ?
+    new URL(`#${slug}`, url).toString() :
+    // from the docs https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/thing_thing.html#getthing
+    `https://inrupt.com/.well-known/sdk-local-node/${slug}`;
```

Comment:
Wild, that's an interesting way on their part to do it. So this definitely works with `getThing` on the local dataset, and it looks like that url is a 404, so I think that means it should be fine if this gets passed to `useThing` as well? It will just assume the thing doesn't exist, which is true, because it hasn't been written yet.

Reply:
yep I think that's right!